<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chatWidget</class>
 <widget class="QWidget" name="chatWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1110</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#nameText, #userName
{
	font-size:28px;
	color: rgba(200, 200, 200, 1);
}
#userName
{
	font-size:22px;
}
#infoText
{
	font-size:15px;
	color: rgba(200, 200, 200, 1);
}</string>
  </property>
  <widget class="QWidget" name="mainWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1110</width>
     <height>700</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
	background-color:rgba(0,0,0,0);
}
#sendButton
{
	background-color: rgb(176, 224, 230);
	font-size:15px;
}</string>
   </property>
   <widget class="QWidget" name="infoWidget" native="true">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="nameText">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>聊天</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QScrollArea" name="chatList">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>250</width>
      <height>590</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget
{
	border-radius: 0px;
	background-color: rgb(150,150,150);
}
QPushButton
{
	border: none;
	font-size: 20px;
	color: rgb(255,255,255);
	text-align:left;
	padding-left:18px;
}
QPushButton:pressed
{
	background: rgb(160,160,160);
}
QScrollBar:vertical
{
	width: 8px;
	background:rgb(130,130,130);
	padding:0px;
	border-radius: 4px;
}
QScrollBar::handle:vertical
{
	background:rgb(180,180,180);
	border-radius: 4px;
}
QScrollBar::handle:vertical:hover
{
	background: rgb(160,160,160);
}
QScrollBar::handle:vertical:pressed
{
	background: rgb(150,150,150);
}
QScrollBar::add-page:vertical
{
	background:none;
}
QScrollBar::sub-page:vertical
{
	background:none;
}
QScrollBar::add-line:vertical
{
	background:none;
}
QScrollBar::sub-line:vertical
{
	background: none;
}</string>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustToContents</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>250</width>
       <height>590</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QScrollArea" name="messageList">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>112</y>
      <width>860</width>
      <height>438</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget
{
	border-radius: 0px;
	background: rgba(0,0,0,0);
}
QLabel
{
	background-color:rgb(100,100,100);
	border-radius:10px;
	font-size:14px;
	padding:10px;
	color: rgb(255,255,255);
}
QScrollBar:vertical
{
	width: 8px;
	background:rgb(130,130,130);
	padding:0px;
	border-radius: 4px;
}
QScrollBar::handle:vertical
{
	background:rgb(180,180,180);
	border-radius: 4px;
}
QScrollBar::handle:vertical:hover
{
	background: rgb(160,160,160);
}
QScrollBar::handle:vertical:pressed
{
	background: rgb(150,150,150);
}
QScrollBar::add-page:vertical
{
	background:none;
}
QScrollBar::sub-page:vertical
{
	background:none;
}
QScrollBar::add-line:vertical
{
	background:none;
}
QScrollBar::sub-line:vertical
{
	background: none;
}</string>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustToContents</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>860</width>
       <height>438</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="nameWidget" native="true">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLabel" name="userName">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
          </font>
         </property>
         <property name="text">
          <string>名字</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="textWidget" native="true">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>550</y>
      <width>861</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget
{
	border:none;
	border-radius: 0px;
}
QTextEdit
{
	color: rgb(255,255,255);
}
QScrollBar:vertical
{
	width: 8px;
	background:rgb(130,130,130);
	padding:0px;
	border-radius: 4px;
}
QScrollBar::handle:vertical
{
	background:rgb(180,180,180);
	border-radius: 4px;
}
QScrollBar::handle:vertical:hover
{
	background: rgb(160,160,160);
}
QScrollBar::handle:vertical:pressed
{
	background: rgb(150,150,150);
}
QScrollBar::add-page:vertical
{
	background:none;
}
QScrollBar::sub-page:vertical
{
	background:none;
}
QScrollBar::add-line:vertical
{
	background:none;
}
QScrollBar::sub-line:vertical
{
	background: none;
}
QPushButton
{
	border-radius: 10px;
	color:rgb(255,255,255);
}</string>
    </property>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>841</width>
       <height>111</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QTextEdit
{
	background-color: rgb(36,36,42);
}
QMenu
{
	border-radius: 10px;
}</string>
     </property>
    </widget>
    <widget class="QPushButton" name="fileButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>3</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::DocumentNew"/>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="videoButton">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>3</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::CameraVideo"/>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="sendButton">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>645</y>
      <width>70</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>550</y>
      <width>861</width>
      <height>2</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget
{
	background-color: rgb(50,50,50);
	border:none;
	border-radius: 0px;
}</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>110</y>
      <width>861</width>
      <height>2</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget
{
	background-color: rgb(50,50,50);
	border:none;
	border-radius: 0px;
}</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
